game/engine.cc

```diff
@@ -25,17 +24,17 @@ Engine::Engine(Audio& audio, VocalTrackPtrs vocals, Database& database):
 		m_database.cur.push_back(Player(*vocals[i], a, frames));
 		++i;
 	}
-	m_thread.reset(new boost::thread(boost::ref(*this)));
+	m_thread.reset(new std::thread(std::ref(*this)));
```

Comment:
This could be changed the same way like the rest, to use make_unique, but I had not enough time to do all of them. Also, make_shared should be used when the result is a shared_ptr (although make_unique will also work because of implicit conversion from unique into shared).